samples/utils/CommandLineUtils.h

```diff
@@ -0,0 +1,139 @@
+#pragma once
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#include <aws/crt/Types.h>
+
+namespace Utils
+{
+    /**
+     * A struct to hold the command line options that can be passed to the program from the terminal/console.
+     */
+    struct CommandLineOption
+    {
+        Aws::Crt::String CommandName;
```

Comment:
It's not done with 100% consistency, but using camel case with a "m_" prefix for member names is usually what you'll see in the codebase, ie. m_commandName